src/adapter/http/src/smart_protocol/axum_server_pull_protocol.rs

```diff
@@ -48,7 +48,7 @@ impl AxumServerPullProtocolInstance {
         let pull_request = match self.handle_pull_request_initiation().await {
             Ok(pull_request) => pull_request,
             Err(e) => {
-                tracing::debug!("Pull process aborted with error: {}", e);
+                tracing::error!("Pull process aborted with error: {}", e);
```

Comment:
If I'm reading this correctly, in case of this and similar errors below we will:
- Log only top-level error message (which could be a non-informative `Internal error`) without a backtrace
- `return` out of the function which will cause socket to close abruptly, not providing any information to the client